src/helpers/currencies/CurrencyURIScheme.js

```diff
@@ -42,10 +43,10 @@ export function decodeURIScheme(str: string): Data {
   const { amount, ...specificFields } = { ...query };
   Object.assign(data, specificFields);
   if (amount) {
-    const amountFloat = parseFloat(amount);
-    if (!isNaN(amountFloat) && amountFloat > 0) {
+    const amountFloat = BigNumber(amount);
+    if (!amountFloat.isNaN() && amountFloat.isPositive()) {
```

Comment:
FIXME isPositive actually includes ZERO. wasn't aware of this

Reply:
yeah i know aha. xD

Reply:
maybe it will just be better to always use `isGreaterThan(0)` because there is also `isGreaterThanOrEqualTo(0)` 
a bit verbose but explicit

Reply:
OH there is also `gt` and `gte` synthax

Reply:
[](https://raw.githubusercontent.com/MikeMcl/bignumber.js/gh-pages/API.png)